### PR TITLE
Full Website for Math Magicians

### PR DESCRIPTION
**Pull request by mrsamsharma**
- Used React Router to route three pages to create SPA
- Implement a good looking UI
- Separated Calculator Component for code quality

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
 import React from 'react';
-import Calculator from './components/Calculator';
+import Home from './pages/Home';
+import Navbar from './components/Navbar';
 
 // eslint-disable-next-line react/prefer-stateless-function
```
Comment (gemmen29) [inline src/App.js]: - [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.

File: src/components/Digit.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prop-types
```
Comment (gemmen29) [inline src/components/Digit.js]: - [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.

File: src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import '../styles.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (gemmen29) [inline src/components/Navbar.js]: - [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.

File: src/components/Operation.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prop-types
```
Comment (gemmen29) [inline src/components/Operation.js]: - [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.

File: src/index.js [hunk 5]
```diff
@@ -1,11 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (gemmen29) [inline src/index.js]: - [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.

File: src/pages/Calculator.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
+import { useState, React } from 'react';
+import '../styles.css';
+import calculate from '../logic/calculate';
+import Digit from '../components/Digit';
+import Operation from '../components/Operation';
+import Navbar from '../components/Navbar';
+
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (gemmen29) [inline src/pages/Calculator.js]: - [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @mrsamsharma, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
Hi @mrsamsharma,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**